src/fpm_settings.f90

```diff
@@ -0,0 +1,231 @@
+!> Manages global settings which are defined in the global config file.
+module fpm_settings
+  use fpm_filesystem, only: exists, join_path, get_local_prefix, is_absolute_path, mkdir
+  use fpm_environment, only: os_is_unix
+  use fpm_error, only: error_t, fatal_error
+  use fpm_toml, only: toml_table, toml_error, toml_stat, get_value, toml_load, check_keys
+  use fpm_os, only: get_current_directory, change_directory, get_absolute_path, &
+                    convert_to_absolute_path
+  implicit none
+  private
+  public :: fpm_global_settings, get_global_settings, get_registry_settings, official_registry_base_url
+
+  character(*), parameter :: official_registry_base_url = 'https://fpm-registry.onrender.com'
```

Comment:
Can we put this base url out side of the source code in some environment variable and read if from there ?

Reply:
You can specify a custom registry in the global config file if that is what you mean

Reply:
Yes, the base url might not be final here, but it is also nothing that should be set by the user. I'm also a bit puzzled whether we should leave it like that. @awvwgk